### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by SamChou19815**
### Summary 

Update docs with the latest architecture information.

### Test Plan 

Read it  👀.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,36 +48,52 @@ Navigate to http://localhost:3000/admin and enter your password when prompted.
 Select the “Initialize Database” button and wait for the database to populate.
 Once this process completes, the application is ready to use!
```
Comment (SamChou19815) [inline README.md:49]: This section is outdated. Guide on how to update it would be appreciated.

Comment (khemritolya) [reply README.md:49]: That button was busted *before* we merged `meteor-shower`. It was on our to-do list for backend fixes. Right now the only way to populate the database is with mongorestore.

Comment (khemritolya) [reply README.md:49]: Something like this dumprestore.sh which restores relevant bson files passed to it from a directory to a local mongodb running at port 3001.
```
# Restore all bson files found in the directory passed in as a command line argument
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/classes.bson" --drop
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/subjects.bson" --drop
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/reviews.bson" --drop
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
-You can access the app at http://localhost:3000.
+```bash
+yarn
+yarn workspace server start # start server
+yarn workspace client start # start client
+```
+
+You can access the app at [http://localhost:3000].
 
 Next, add a password to your local database. Launch a second terminal window and
```
Comment (khemritolya) [inline README.md:38]: I should mention that this is not how we do admin validation anymore. We set field on the user's mongodb entry.

File: README.md (hunk 1 above)
Comment (SamChou19815) [reply README.md:49]: @khemritolya since you are more knowledge on this, could you help fix this part of the docs?

**Comment by Urist1** [comment]
I made my changes @SamChou19815 . Let me know what you think.

**Comment by khemritolya** [comment]
I made the changes @SamChou19815 . Let me know what you think.

**Comment by SamChou19815** [comment]
@khemritolya looks good. You can hit approve and merge it

**Pull request merged**